R/statistical_tests.R

```diff
@@ -121,7 +119,7 @@ chisq.test.tabyl <- function(x, tabyl_results = TRUE, ...) {
 
 
 
-#' @title Apply stats::fisher.test to a two-way tabyl
+#' Apply stats::fisher.test to a two-way tabyl
```

Comment:
Do you want to make this `stats::fisher.test` to make it consistent with the formatting change above to `stats::chisq.test` ?

Reply:
I agree!